lib/presentation/widget/shared_space/file_surfing/workgroup_nodes_surfing_widget.dart

```diff
@@ -580,4 +584,56 @@ class _WorkGroupNodesSurfingWidgetState extends State<WorkGroupNodesSurfingWidge
       ),
     );
   }
+
+  Widget _buildMenuSorter() {
```

Comment:
I think this widget should be written into a usable Widget. Because of it's same to [my_space_widget](https://github.com/linagora/linshare-mobile-flutter-app/blob/9576803f90d541bb8dd16cc8d8f5b6eb40cd301c/lib/presentation/widget/myspace/my_space_widget.dart#L634)

Reply:
cool, i try it